### PR TITLE
chore: Improve onboarding docs, fix docs build [Canary]

### PR DESCRIPTION
**Pull request by wodenx**
## Changes

Complete the vital onboarding guide introductory lessons
Add `withChildNode` to create a node + nodeKey
Set a maximum depth for auto-discovery of bodiless docs
Rename vital-examples to @bodiless namespace
Remove multilingual activation from starter sites (now commented out)

### CODE REVIEW

File: packages/bodiless-layouts/package.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
     "access": "public"
   },
   "scripts": {
-    "build": "run-p build:lib && npm run build:api-doc",
+    "build": "run-p build:lib",
```
Comment (dewen) [inline packages/bodiless-layouts/package.json:30]: restore the ` && npm run build:api-doc`?

File: packages/bodiless-i18n/package.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
     "access": "public"
   },
   "scripts": {
-    "build": "npm run build:lib && npm run build:api-doc",
+    "build": "npm run build:lib",
```
Comment (dewen) [inline packages/bodiless-i18n/package.json:23]: restore `build:api-doc`?

File: packages/vital-examples/doc/README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# Vital Design System Curriculum
+
+Welcome to the VitalDS learning curriculum. Here you will find what you need to get started working
+with Vital and Bodiless.
+
+## Before you start
+
+To help you nderstand some of the foundational ideas of VitalDS and BodilessJS, and the problems they
```
Comment (dewen) [inline packages/vital-examples/doc/README.md:8]: => understand

File: packages/vital-examples/doc/README.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+https://www.youtube.com/watch?v=_LBdqpscwi0)
+
+
+## Important Notes
+
+- You will need a local development environment with NodeJS v18.16 or higher.
+- The working code for all these examples can be found in the
+  [@bodiless/vital-examples](https://github.com/johnsonandjohnson/Bodiless-JS/tree/main/packages/vital-examples)
+  package and in the [vital-examples site](https://github.com/johnsonandjohnson/Bodiless-JS/tree/main/sites/vital-examples).
+- All components and tokens created in the examples are structured accoring to very specific
```
Comment (dewen) [inline packages/vital-examples/doc/README.md:25]: => according

File: packages/vital-examples/doc/README.md (hunk 3 above)
Comment (dewen) [reply packages/vital-examples/doc/README.md:8]: included in another pr

File: packages/vital-examples/doc/README.md (hunk 4 above)
Comment (dewen) [reply packages/vital-examples/doc/README.md:25]: fixed in another pr

**Review by dewen — APPROVED** [review]

**Comment by wodenx** [comment]
## Changes

Complete the vital onboarding guide introductory lessons
Add `withChildNode` to create a node + nodeKey
Set a maximum depth for auto-discovery of bodiless docs
Rename vital-examples to @bodiless namespace
Remove multilingual activation from starter sites (now commented out)

**Pull request merged**